gnomad/utils/vcf.py

```diff
@@ -683,7 +683,7 @@ def make_combo_header_text(
     Programmatically generate text to populate the VCF header description for a given variant annotation with specific groupings and subset.
 
     For example, if preposition is "for", group_types is ["group", "pop", "sex"], and combo_fields is ["adj", "afr", "female"],
-    this function will return the string " for female samples of African-American/African ancestry".
+    this function will return the string " for female samples in the African-American/African genetic ancestry group".
```

Comment:
nitpick, but we should also update gender terms to sex karyotypes in our docstrings

Reply:
I made https://app.zenhub.com/workspaces/gnomad-5f4d127ea61afc001d6be50b/issues/gh/broadinstitute/gnomad_production/1475 so this is tackled post 4.1